templates/zerver/help/footer.md

```diff
@@ -0,0 +1,7 @@
+---
+
+Click [here](/help/) to go back to the documentation home page.
+
+This user documentation is open source which means that anyone can contribute
```

Comment:
Shouldn't we mention a specific licence it is under, isn't just saying open source a bit ambiguous?